go.mod

```diff
@@ -32,3 +32,5 @@ require (
 	golang.org/x/sync v0.6.0 // indirect
 	golang.org/x/sys v0.22.0 // indirect
 )
+
+replace github.com/notaryproject/notation-go => github.com/Two-Hearts/notation-go v0.0.0-20240815061530-d5e876aecb1d
```

Comment:
Leave a comment as a reminder for dependency update.

Reply:
Updated.